perf/semgrep-core-benchmark

```diff
@@ -0,0 +1,193 @@
+#! /usr/bin/env python3
+#
+# Run semgrep on a series of pairs (rules, repo) with different options,
+# and report the time it takes. Optionally upload the results to the semgrep
+# dashboard.
+#
+# With the -semgrep-core flag, run semgrep-core with different options and
+# report the time it takes. Requires semgrep-core (make install in the
+# semgrep-core folder)
```

Comment:
Is there a `-semgrep-core` flag or is it an idea you dropped? Can you explain why there's this new script for semgrep-core and another for semgrep (`run-benchmarks`)? Can we have just one?

Reply:
Sorry, this is an idea I dropped. I thought I discarded the changes, but I should have checked. I decided to add a new script because I needed an additional field for the config files, -lang. Not all the config files would really have been useable with semgrep-core, so it seemed too complicated to put the two in the same file

Reply:
The reason why I think we should have benchmarks for `semgrep` is that it may scan files differently or do some weird things that impact performance. But it really shouldn't do weird things, and it should allow us to do everything that semgrep-core allows.

A user on community Slack was asking for a `--lang` filter that works on top of the semgrep rules file. Would it help here?